### PR TITLE
chore: remove spammy debug statements

### PR DESCRIPTION
**Pull request by mcmxcdev**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/HEAD/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

#### Description of Change
- remove various `debug()` invocations

#### Motivation and Context
I haven't seen applications keep around so many debug statements inside the production code like Ferdium does. In my opinion, the application should be debugged in dev environment and otherwise be clean. 

Now we have less lines of code, less logs fighting for attention and its clearer to see when relevant warnings and errors pop up.

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`pnpm prepare-code`)
- [x] `pnpm test` passes
- [x] I tested/previewed my changes locally

### CODE REVIEW

File: src/webview/lib/RecipeWebview.ts [hunk 1]
```diff
@@ -30,8 +30,6 @@ class RecipeWebview {
     ipcRenderer.on('poll', () => {
       this.loopFunc();
 
-      debug('Poll event');
```
Comment (mcmxcdev) [inline src/webview/lib/RecipeWebview.ts:33]: This was probably the worst offender which made it impossible to look at logs in an unfiltered state inside the dev tools.

File: src/stores/RecipesStore.ts [hunk 2]
```diff
@@ -85,9 +85,6 @@ export default class RecipesStore extends TypedStore {
     const recipeIds = this.recipeIdForServices;
     const recipes = {};
 
-    // Hackfix, reference this.all to fetch services
-    debug(`Check Recipe updates for ${this.all.map(recipe => recipe.id)}`);
```
Comment (vraravam) [inline src/stores/RecipesStore.ts:89]: removing this might invalidate that HACKFIX

**Review by vraravam — COMMENTED** [review]
I can see the value in decluttering the logs with the poll event, but, imo some of the other debug stmts are valid. I'm not sure if its a good idea to remove all of them en masse. I'll leave it upto the other contributors to chime in on the value

**Comment by gogosese** [comment]
Ferdium is really powerful, and I am still learning it. I am still looking for solutions when I encounter problems. When I was chatting, Google Translate couldn't be used normally.

**Comment by mcmxcdev** [comment]
Any opinions on this one @Alphrag and @SpecialAro ? If it's not desired to clean up the logs, then I can close this PR.

**Comment by mcmxcdev** [comment]
I will close this since there seems to be no consensus that this is what we want.

**Pull request closed without merging**